Orange/widgets/utils/__init__.py

```diff
@@ -1,13 +1,16 @@
 from functools import reduce
+from Orange.data.variable import TimeVariable
 
 
 def vartype(var):
     if var.is_discrete:
         return 1
-    elif var.is_continuous:
+    elif var.is_continuous and not isinstance(var, TimeVariable):
         return 2
     elif var.is_string:
         return 3
+    elif isinstance(var, TimeVariable):
+        return 4
```

Comment:
Not really important, but:

If we want this if-else flat and ordered, then I am in favour of changing it to go from 4 to 0.
TimeVariable (and future new types) can be added to the top and any unhandled variables are caught at the bottom as generic variable types.

Another option would be a nested if-else with all continuous types added inside the elif is_continuous. This shows the structure better, but loses the ordering and becomes more complex.